grpc/grpc_server.cpp

```diff
@@ -0,0 +1,83 @@
+#include "requests_common.hpp"
+
+#include "read_request.hpp"
+#include "write_request.hpp"
+
+namespace fb = flatbuffers;
+
+namespace grpc_dnet {
+
+struct server {
+public:
+	server(dnet_node *node)
+		: node_(node)
```

Comment:
Could you please adjust code style in cases like this to:
```suggestion
	: node_(node)
```

Reply:
Will fix.

I noticed that in out-of-class constructor implementations initializer list is written with tab: https://github.com/interiorem/elliptics/blob/e5c8cbece648a33b8650a0f3adcb0ba9ddb3bd92/library/protocol.cpp#L544
Should I use tab in implementations?

Reply:
No, the style should be the same.